protocol/messenger.go

```diff
@@ -1234,14 +1234,11 @@ func (m *Messenger) Contacts() []*Contact {
 }
 
 // GetContactByID assumes pubKey includes 0x prefix
-func (m *Messenger) GetContactByID(pubKey string) (*Contact, error) {
+func (m *Messenger) GetContactByID(pubKey string) *Contact {
 	m.mutex.Lock()
 	defer m.mutex.Unlock()
-	contact, ok := m.allContacts[pubKey]
-	if !ok {
-		return nil, errors.New("no contact found")
-	}
-	return contact, nil
+
+	return m.allContacts[pubKey]
```

Comment:
So why exactly are we dropping the error here?

Reply:
Before we returned error if the contact was not in memory (i.e not in database), desktop prefers not to return an error in this case, and just return `null`, and it's only used by desktop

Reply:
Makes sense.